sw/airborne/subsystems/abi_sender_ids.h

```diff
@@ -162,12 +162,16 @@
 #define GPS_SIM_ID 11
 #endif
 
+#ifndef GPS_IMCU_ID
```

Comment:
Would be better to add a new ID 14 instead of changing the IDs of existing ones...